website/docs/r/organization.html.markdown

```diff
@@ -0,0 +1,39 @@
+---
+layout: "tfe"
+page_title: "Terraform Enterprise: tfe_organization"
+sidebar_current: "docs-tfe-organization"
+description: |-
+  Creates, updates and destroys organizations.
+---
+
+# tfe_organization
+
+Creates, updates and destroys organizations.
```

Comment:
`Creates, updates and destroys` -> `Manages`

Reply:
Fixed in next commit.